src/plugin/libreofficetoolkit-qml-plugin/LibreOfficeKit/LibreOfficeKitInit.h

```diff
@@ -203,23 +209,43 @@ static LibreOfficeKit *lok_init_2( const char *install_path,  const char *user_p
     imp_lib = strdup("the app executable");
     dlhandle = RTLD_MAIN_ONLY;
 #endif
+    *_imp_lib = imp_lib;
+    return dlhandle;
+}
+
+typedef LibreOfficeKit *(LokHookFunction)( const char *install_path);
```

Comment:
Here (and in the lines below) too, please remove spaces after `(` and before `)`